docs/proposals/multi-node/README.md

```diff
@@ -987,12 +1234,21 @@ Hence, it is better to keep `etcd-druid` altogether agnostic of issues related t
 
 ### PodDisruptionBudget
 
-This proposal recommends that `etcd-druid` should deploy [`PodDisruptionBudget`](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/#pod-disruption-budgets) (`maxUnavailable` set to `floor(<cluster size>/2)`) for multi-node etcd clusters to ensure that any planned disruptive operation can try and honour the disruption budget to ensure high availability of the etcd cluster.
+This proposal recommends that `etcd-druid` should deploy [`PodDisruptionBudget`](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/#pod-disruption-budgets) (`minAvailable` set to `floor(<cluster size>/2) + 1`) for multi-node etcd clusters (if `AllMembersReady` [condition](#conditions) is `true`) to ensure that any planned disruptive operation can try and honour the disruption budget to ensure high availability of the etcd cluster while making potentially disrupting maintenance operations.
+
+In addition, it is recommended to toggle the `minAvailable ` field between `floor(<cluster size>/2) + 1` and `<cluster size>` whenever the `BackupReady` condition toggles between `true` and `false`.
+This is to disable eviction of member pods when backups are not healthy.
+
+Also, it is recommended to toggle the `minAvailable` field between `floor(<cluster size>/2)` and `<number of members with status Ready true>` whenever the `AllMembersReady` condition toggles between `true` and `false`.
+This is to disable eviction of any member pods when not all members are `Ready`.
```

Comment:
It is somewhat annoying in Kubernetes that we have to do that via our control loops, i.e. PDBs do not take the readiness probe into account when counting what is "down".

Reply:
Yes. It would have been ideal if the PDBs were automatically adjusted based on `readyReplicas`.